src/inabox/amp-inabox.js

```diff
@@ -81,11 +82,11 @@ startupChunk(self.document, function initial() {
   perf.tick('is');
 
   self.document.documentElement.classList.add('i-amphtml-inabox');
-  // TODO(lannka): remove ampDocCss for inabox rendering #22418
-  const fullCss =
-    ampDocCss +
-    ampSharedCss +
-    'html.i-amphtml-inabox{width:100%!important;height:100%!important}';
+  const fullCss = isExperimentOn(self, 'inabox-css-cleanup')
+    ? ampSharedCss
+    : ampDocCss +
+      ampSharedCss +
+      'html.i-amphtml-inabox{width:100%!important;height:100%!important}';
```

Comment:
The last part is a big jump, imho. This last CSS line was meant specifically for in-a-box. Are you comfortable to remove both "ampdoc" part and the last "inabox" part? I feel like it'll definitely cause some issues. Maybe we should start with removing "!important"?

Reply:
moved this out from this experiment.